MSAL/src/MSALErrorConverter.m

```diff
@@ -201,6 +212,7 @@ + (NSError *)errorWithDomain:(NSString *)domain
         msalUserInfo[mappedKey] = userInfo[key];
     }
 
+    if (!msalUserInfo[MSALCorrelationIDKey] && correlationId) msalUserInfo[MSALCorrelationIDKey] = correlationId.UUIDString;
```

Comment:
I believe we can now remove __unused tag above

Reply:
Do you mean the method signature? If yes, I will remove it and make sure the type is correct